docs/en/stack/ml/anomaly-detection/index.asciidoc

```diff
@@ -28,68 +26,72 @@ include::stopping-ml.asciidoc[leveloffset=+2]
 
 include::ml-restart-failed-jobs.asciidoc[leveloffset=+2]
 
+include::ml-ad-concepts.asciidoc[leveloffset=+1]
+
 include::anomaly-detection-scale.asciidoc[leveloffset=+2]
 
 include::ml-api-quickref.asciidoc[leveloffset=+1]
 
-include::ootb-ml-jobs.asciidoc[leveloffset=+1]
+include::anomaly-examples.asciidoc[leveloffset=+1]
 
-include::ootb-ml-jobs-apache.asciidoc[leveloffset=+2]
+include::{es-repo-dir}/ml/anomaly-detection/ml-configuring-alerts.asciidoc[leveloffset=+2]
```

Comment:
Would it be worth it to still keep these files in the ES repo? Is it feasible to move them in the stack-docs repo in a separate PR after the reorg is done? So all of the AD-related conceptual docs files would be in the same place.

Reply:
It's definitely something to consider, but I'll defer that decision for now.